src/sonarwhal-theme/layout/partials/home.hbs

```diff
@@ -22,86 +13,117 @@
-                            <p class="feature_highlight">
-                                With the help of our <a href="/docs/contributor-guide/">developer guide</a>, you can
-                                <a href="/docs/contributor-guide/rules/index.html">create new rules</a> to suit <em>your</em> needs. You can
-                                help sonarwhal help even more people like you by contributing your rules back!
-                            </p>
-                        </div>
-                        <div class="module">
-                            <p class="subtitle--alt subtitle--home uppercase-text">Community driven</p>
-                            <p class="feature_highlight">
-                                sonarwhal welcomes anyone who wants to make the web a better place.
-                                Testing, <a href="https://github.com/sonarwhal/sonarwhal.com/issues">filing issues and feature requests</a>,
-                                <a href="/docs/developer-guide/contributing/pull-requests.html">contributing code</a>, and improving the documentation
-                                are just the start!
-                            </p>
-                        </div>
-                    </div>
-                </section>
+                    <p class="container--section__welcome">sonarwhal is a linting tool that will help you with your site's accessibility, speed, security and more, by checking your code for best practices and common errors.</p>
+                </div>
+                <div class="module module--secondary">
+                    <img src="/images/home-hello-nellie.svg" alt="Nellie the Narwhal" class="hello-illustration" />
+                </div>
             </div>
-            <div class="dark-blue">
-                <section class="home-container home-container--section home-container__install">
-                    <h2 class="subheading subheading--home">Install sonarwhal</h2>
-                    <div class="layout layout--halves vertical-align">
-                        <div class="module">
-                            <img src="/images/home-nellie.svg" class="sub-illustration" alt="" role="presentation">
+        </section>
+        <div class="bg-wrap--white">
+            <section class="container home-container--section home-container__features">
+                <h2 class="subheading subheading--home">Why use sonarwhal?</h2>
+                <div class="layout layout--thirds">
+                    <div class="module">
+                        <img src="/images/nellie-customizable.svg" alt="" role="presentation" />
+                        <p class="subtitle subtitle--home uppercase-text">Fully customizable</p>
+                        <p class="feature__highlight">
+                            Every site is different. sonarwhal adapts its feedback when you give it more information:
+                            <a href="/docs/user-guide/#ignoringdomains">ignore 3rd-party code</a>,
```

Comment:
This should be: https://sonarwhal.com/docs/user-guide/further-configuration/ignoring-domains/